projects/swarmd/swarmd.yml

```diff
@@ -29,27 +31,16 @@ services:
   - name: ntpd
     image: "linuxkit/openntpd:45deeb05f736162d941c9bf494983f655ab80aa5"
   - name: swarmd
-    image: "linuxkit/swarmd:a2f57f14f07fb6d7cded7832b2dabe878b28554e"
+    image: "linuxkit/swarmd:fa2772d78255d6868bbe8910bcca3cedbb067bf8"
     command: ["/usr/bin/swarmd", "--containerd-addr=/run/containerd/containerd.sock", "--log-level=debug", "--state-dir=/var/lib/swarmd"]
     capabilities:
-     - CAP_CHOWN
-     - CAP_DAC_OVERRIDE
-     - CAP_FSETID
-     - CAP_FOWNER
-     - CAP_MKNOD
-     - CAP_NET_RAW
-     - CAP_SETGID
-     - CAP_SETUID
-     - CAP_SETFCAP
-     - CAP_SETPCAP
-     - CAP_NET_BIND_SERVICE
-     - CAP_SYS_CHROOT
-     - CAP_KILL
-     - CAP_AUDIT_WRITE
+     - all
     pid: host
```

Comment:
With CNI networking you need to be able to construct `/proc/PID/ns/net` to pass to the CNI plugin which is forked by swarmd (and hence in the same namespace as swarmd), but `PID` is from containerd's pid namespace since it is what is launching the containers so the swarmd and containerd pid namespaces must be the same. Since I'm still using the host containerd that means using pid=host.

Eventually I want to run my own containerd instance for swarmd, which will likely change this.